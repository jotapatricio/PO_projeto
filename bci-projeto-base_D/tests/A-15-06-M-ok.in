6
4
qwerweqrw
4
Joao
4
Fonseca
4
 Fonseca
0
0
