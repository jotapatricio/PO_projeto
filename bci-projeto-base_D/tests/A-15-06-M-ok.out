Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Introduza o identificador do criador: Mostrar todas as obras de um criador: Operação inválida: O criador 'qwerweqrw' não existe.
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Introduza o identificador do criador: Mostrar todas as obras de um criador: Operação inválida: O criador 'Joao' não existe.
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Introduza o identificador do criador: Mostrar todas as obras de um criador: Operação inválida: O criador 'Fonseca' não existe.
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Introduza o identificador do criador: Mostrar todas as obras de um criador: Operação inválida: O criador ' Fonseca' não existe.
Menu de obras
1 - Mostrar obra
2 - Mostrar todas as obras
3 - Efectuar pesquisa de termos
4 - Mostrar todas as obras de um criador
5 - Alterar inventário de uma obra
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir ficheiro
2 - Guardar ficheiro
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 